docs/interfaces/_index_.pluginconfig.md

```diff
@@ -40,10 +42,10 @@ PluginConfig is used to provide multiple replacement.
 
 ## Properties
 
-###  replacements
+### replacements
 
-• **replacements**: *[Replacement](_index_.replacement.md)[]*
+•  **replacements**: [Replacement](_index_.replacement.md)[]
 
-*Defined in [index.ts:93](https://github.com/google/semantic-release-replace-plugin/blob/master/src/index.ts#L93)*
+*Defined in [index.ts:93](https://github.com/Borduhh/semantic-release-replace-plugin/blob/d2e8b02/src/index.ts#L93)*
```

Comment:
revert link to be able to merge

Reply:
My mistake, I ran `npm run all`, and that seemed to be a result. Should be all set.